src/flex/FlexView.js

```diff
@@ -76,6 +73,17 @@ const PropTypes = {
 @props(PropTypes, { strict: false })
 export default class FlexView extends React.Component {
 
+  componentDidMount() {
+    this.logWarnings();
+  }
+
+  logWarnings = () => {
+    const { auto } = this.props;
+    if (!t.Nil.is(auto)) {
+      warn('"auto" is DEPRECATED. Use combination of "grow", "shrink" and "basis" instead');
```

Comment:
we are not deprecating `auto` with this change, we are completely removing it :P

So this should be changed in something more strong, but probably could just be listed as breaking change in our CHANGELOG